tasks/main.yml

```diff
@@ -46,3 +46,7 @@
 
 # Update Jenkins and install configured plugins.
 - include: plugins.yml
+
+# Remove init.groovy.d/basic-security settings
+- name: Remove Jenkins security init scripts
+  shell: "rm -f /var/lib/jenkins/init.groovy.d/basic-security.groovy"
```

Comment:
I think this task is causing the idempotence test failure on Travis. It should probably be changed to remove the file using the [file module](http://docs.ansible.com/ansible/file_module.html).